document/core/appendix/algorithm.rst

```diff
@@ -25,10 +25,10 @@ Types are representable as an enumeration.
 
 .. code-block:: pseudo
 
-   type val_type = I32 | I64 | F32 | F64 | Funcref | Externref
+   type val_type = I32 | I64 | F32 | F64 | V128 | Funcref | Externref
 
    func is_num(t : val_type | Unknown) : bool =
```

Comment:
not related to this PR, do we want to update this algorithm to have `is_vec`?

Reply:
Yes! In fact, `is_num` is incorrect as is, and once fixed, `is_vec` is needed in the case for `select`. Probably my oversight, but would you be interested in creating a PR against the main repo?

Reply:
Done, ptal https://github.com/WebAssembly/spec/pull/1399